roles/confluent.zookeeper/defaults/main.yml

```diff
@@ -11,6 +11,7 @@ zookeeper:
     autopurge.snapRetainCount: 10
     autopurge.purgeInterval: 1
     dataDir: /var/lib/zookeeper
+    myid: 1
```

Comment:
I don't think that this variable should have a default value, this bears the risk of duplicate ids if you forget specifying an id.